context/app/static/js/shared-styles/dropdowns/IconDropdownMenu/style.ts

```diff
@@ -0,0 +1,26 @@
+import SvgIcon from '@mui/material/SvgIcon';
+import Typography from '@mui/material/Typography';
+import { styled } from '@mui/material/styles';
+
+import { SecondaryBackgroundTooltip } from 'js/shared-styles/tooltips';
+// TODO: Would it be ok to create visualization/style.js and put this there,
+// so all the buttons would use it?
+// Individual buttons padding on the left and right seems fragile, if they get rearranged.
+// export const StyledSecondaryBackgroundTooltip = styled(SecondaryBackgroundTooltip)`
+//   marginLeft: theme.spacing${(props) => props.theme.spacing(1)};
+// `;
+export const StyledSecondaryBackgroundTooltip = styled(SecondaryBackgroundTooltip)(({ theme }) => ({
+  marginLeft: theme.spacing(1),
+}));
+
+export const StyledSvgIcon = styled(SvgIcon)({
+  height: '20px',
+  width: '20px',
```

Comment:
`fontSize: 1.25rem` should work. We've been somewhat consistent about using `rem` for styling icons.